new-website/utils/tutorials/build_pdf_book.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
@GreatRSingh Add a section to the documentation / README on the PDF building capability